packages/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js

```diff
@@ -8,25 +8,435 @@
+      
+      const style = keysToAnimate.reduce((prev, key) => {
+        const animatedNodeKey = styleParent.config.style[key];
+        const animatedNode = animatedNodes[animatedNodeKey]
+        const value = traverseValueNode(animatedNode, { [nodeTag]: animatedValue})
+        return { ...prev, [key]: '"' + value + '"' }
+      }, {})
+      
+      return [{ viewTag: child.viewTag, style }, ...found]
+    }
+    else if(child.children && child.children.length > 0) {
+      return child.children.reduce(viewReducer, found)
+    }
+    return found;
+  };
+
+  const viewTags = animatedNode.children.reduce(viewReducer, [])
+
+  viewTags.forEach(({viewTag, style })=>{
+    Object.keys(styleProp => {
+      viewTag.style[styleProp] = style[styleProp]
+    })
+  });
+}
+
+function times(n, iteratee) {
+  if (n < 1 || n > MAX_SAFE_INTEGER) {
+    return []
+  }
+  let index = -1
+  const length = Math.min(n, MAX_ARRAY_LENGTH)
+  const result = new Array(length)
+  while (++index < length) {
+    result[index] = iteratee(index)
+  }
+  index = MAX_ARRAY_LENGTH
+  n -= MAX_ARRAY_LENGTH
+  while (++index < n) {
+    iteratee(index)
+  }
+  return result
+}
+
+const NativeAnimatedModule = {
+  startOperationBatch: () => {
+    console.warn('startOperationBatch not implemented on web')
+  },
+  finishOperationBatch: () => {
+    console.warn('finishOperationBatch not implemented on web')
+  },
+  createAnimatedNode: (tag: number, config: AnimatedNodeConfig) => {
+    animatedNodes[tag] = { config, children: [], parents: [], tag };
+  },
+  getValue: (tag: number, saveValueCallback: SaveValueCallback) => {
+    const node = animatedNodes[tag];
+    const value = traverseValueNode(node, {})
+    saveValueCallback(value);
+    return value;
+  },
+  startListeningToAnimatedNodeValue: (tag: number) => {
+    console.warn('startListeningToAnimatedNodeValue not implemented on web', {tag})
+  },
+  stopListeningToAnimatedNodeValue: (tag: number) => {
+    console.warn('stopListeningToAnimatedNodeValue not implemented on web', {tag})
+  },
+  connectAnimatedNodes: (parentTag: number, childTag: number) => {
+    if(animatedNodes[parentTag]?.children){
+      animatedNodes[parentTag].children.push(childTag)
+    }
+    if(animatedNodes[childTag]?.parents){
+      animatedNodes[childTag].parents.push(parentTag)
+    }
+  },
+  disconnectAnimatedNodes: (parentTag: number, childTag: number) => {
+    if(animatedNodes[parentTag]?.children){       
+      animatedNodes[parentTag].children = animatedNodes[parentTag].children.filter(c => c !== childTag)
+    }
+    if(animatedNodes[childTag]?.parents){
+      animatedNodes[childTag].parents = animatedNodes[childTag].parents.filter(c => c != parentTag)
+    }
+  },
+  startAnimatingNode: (
+    animationId: string,
+    nodeTag: number,
+    config: AnimatingNodeConfig,
+    endCallback: EndCallback,
+  ) => {
+    //console.log('startAnimatingNode', { nodeTag, config, animatedNodes })
```

Comment:
There's a fair few comments or debug code still present. Cleaning this up / doing a self-review would be helpful to get it merged!